src/helpers/databaseHelpers.ts

```diff
@@ -0,0 +1,113 @@
+import supabase from '@/utils/supabase-client';
+
+interface ScoreRecord {
+  address: string;
+  score?: number;
+  meta?: any;
```

Comment:
Let's avoid using `any`s and properly type our codebase - it might be more work now, but will save a lot of time when we're refactoring things post-launch.